datahub-web-react/src/app/entity/shared/containers/profile/utils.ts

```diff
@@ -107,3 +118,42 @@ export function useEntityQueryParams() {
 
     return response;
 }
+
+export function getOnboardingStepIdsForEntityType(entityType: EntityType): string[] {
+    switch (entityType) {
+        case EntityType.Chart:
+            return [
+                ENTITY_PROFILE_DOCUMENTATION_ID,
+                ENTITY_PROFILE_PROPERTIES_ID,
+                ENTITY_PROFILE_LINEAGE_ID,
```

Comment:
so we're down to show this even if the tab is disabled right? i think that's fine, but the copy might need to reflect that...

Reply:
Yeah copy should say, if it's disabled, X has not been ingested for this entity.

Reply:
Added these notes for lineage and documentation, the only tabs for which I think this is currently relevant